.drone.yml

```diff
@@ -15,6 +15,16 @@ steps:
   - go test ./...
   - sh scripts/build.sh
 
+- name: publish (dry run)
```

Comment:
Please remove this since docker pull of base image fails due to dockerhub rate limit.